File: HorseRacing.Contracts/Models/Game/Dtos/GameDeckCardDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.Game.Dtos
{
    public class GameDeckCardDto : BaseDto
    {
        /// <summary>
        /// Масть
        /// </summary>
        public int CardSuit { get; set; }
        /// <summary>
        /// Номинал
        /// </summary>
        public int CardRank { get; set; }
        /// <summary>
        /// Позиция в колоде
        /// </summary>
        public int CardOrder { get; set; }
        /// <summary>
        /// Нахождение в игре
        /// </summary>
        public int Zone { get; set; }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/dto/game-deck-card-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';
import type { ZoneType } from '../enums/zone-type-enum';
import type { SuitType } from '../enums/suit-type-enum';
import type { RankType } from '../enums/rank-type-enum';

export interface GameDeckCardDto extends BaseDto {
    /** Масть */
    CardSuit: SuitType;
    /** Номинал */
    CardRank: RankType;
    /** Позиция в колоде */
    CardOrder: number;
    /** Нахождение в игре */
    Zone: ZoneType;
}
